doc/atcommands.md

```diff
@@ -62,29 +64,29 @@ Server Uptime: 3 days, 8 hours, 6 minutes, 4 seconds.
 
 ---------------------------------------
 
-@refresh
-@refreshall
+`@refresh
+@refreshall`
```

Comment:
This should be in code block (triple back-tick), so it appears in multiple line
Example:
    ```
    @refresh
    @refreshall
    ````

Reply:
fixed.